arquillian/RESTEASY-1056-jetty-bv11/src/test/resources/arquillian.xml

```diff
@@ -4,8 +4,8 @@
    xsi:schemaLocation="http://jboss.org/schema/arquillian
         http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
 
-   <!-- Force the use of the Servlet 3.0 protocol with all containers, as it is the most mature -->
-   <defaultProtocol type="Servlet 3.0" />
+   <!-- Force the use of the Server 5.0 protocol with all containers, as it is the most mature -->
+   <defaultProtocol type="Server 5.0" />
```

Comment:
You mean "servlet"? According to https://medium.com/swlh/testing-jakarta-ee-9-components-with-arquillian-and-weld-f9e1add08895,

>By default, when arquillian-protocol-servlet-jakarta is ocurred in the classpath, the Servlet 5.0 protocol will be the default protocol, if you have declared the protocol in the existing arquillian.xml file, change it to use Servlet 5.0.

Reply:
Nice catch! I did indeed mean "Servlet 5.0".